src/hooks/useOrders.ts

```diff
@@ -110,14 +115,21 @@ export function useOrders(): Result {
 
   // allow to fresh start/refresh on demand
   const forceOrdersRefresh = useCallback((): void => {
-    setOffset(0)
+    dispatch(updateOffset(0))
     setIsLoading(true)
-  }, [setIsLoading, setOffset])
+  }, [dispatch, setIsLoading])
+
+  const runEffect = useRef(false)
 
   useEffect(() => {
+    if (!runEffect.current) return
```

Comment:
I thought about it and a simpler approach would be
```ts
if (!runEffect.current) {
  runEffect.current = true
  return
}
```

instead of an extra `useEffect`
Sorry about that

Reply:
Unforgivable!

Reply:
Done